src/OpenLoco/Window.h

```diff
@@ -34,18 +34,18 @@ namespace OpenLoco::Ui
         wt_4,
         wt_5,
         wt_6,
-        wt_7,
-        wt_8,
-        wt_9,
-        wt_10,
-        wt_11, // button
+        toolbarTab = 7,
+        tab = 8,
+        buttonWithImage = 9,
+        buttonWithColour = 10,
+        button = 11,
         wt_12,
         wt_13,
-        wt_14,
+        buttonGridSort = 14,
```

Comment:
Not convinced `buttonGridSort` is a clear name, to be honest. I'd prefer something like `tableHeader`.

Reply:
Hmm, don't really mind either way. The actual drawing of the sort chevron is handled by the window rather than the widget which means it doesn't have to be for sorting. But every single user of the button is for sorting at the top of a table.